src/CommunityToolkit.Maui/PopupService.cs

```diff
@@ -62,7 +62,7 @@ public PopupService()
 
 		var popup = GetPopup(typeof(TViewModel));
 
-		ValidateBindingContext<TViewModel>(popup, out _);
+		popup.BindingContext = viewModel;
```

Comment:
Hi @GuidoNeele! As [mentioned in my comment on the corresponding Issue](https://github.com/CommunityToolkit/Maui/issues/1524#issuecomment-1812965568), it is out-of-scope for the method `ShowPopup()` to be assigning / overwriting the values of the Popup's properties.

This could cause bugs in other developer's code who may not want or expect this behavior.

Reply:
I somehow believe it was THE intent of the method to assign the provided viewModel as BindingContext. How else are you supposed to bind it otherwise?